openaq_api/openaq_api/v3/routers/owners.py

```diff
@@ -76,15 +94,43 @@ async def owners_get(
 async def fetch_owners(query, db):
     query_builder = QueryBuilder(query)
     sql = f"""
-    SELECT e.entities_id AS id
-    , e.full_name AS name
-    , COUNT(sn.owner_entities_id) AS locations_count
-    FROM entities e
-    JOIN sensor_nodes sn ON e.entities_id = sn.owner_entities_id
-    {query_builder.where()}
-    GROUP BY e.entities_id, name
-    ORDER BY e.entities_id
+    WITH Owners AS (
+        SELECT 
+            (owner->>'id')::int AS owner_id
```

Comment:
since youre aliasing this anyways i would just use "id" and "name" as the alias so that you dont have to use a field alias in the OwnerBase response model

Reply:
Good idea